peer-test-suite/python/logic.py

```
@@ -1,3 +1,16 @@
-def get_user_age(user):
-    # missing None checks; assume user has 'age'
-    return 2025 - user['birth_year']
+
+def analyze_data(data, cache=[]):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)